samples/bot-release-management/nodejs/aad.manifest.json

```diff
@@ -0,0 +1,123 @@
+{
```

Comment:
Do this project use SSO?

Reply:
No, but this samples required application permissions
![image](https://github.com/OfficeDev/Microsoft-Teams-Samples/assets/109147706/0aedbd42-cb9b-40ce-9130-ffc0e6d01844)